packages/salesforcedx-webview-ui/src/entries/ManifestEditor/index.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
```

Comment:
We should change this to import { render } from 'react-dom'; so it will only import the render method and nothing else.

Reply:
Doesn't the tree shaking and dead-code elimination from webpack solve the issue of bringing too much? Or am I overestimating what it can do?

Reply:
Tree shaking will remove that, but I didn't see anything for that on the webpack config files. Is babel compiling to ES6? If it is then webpack will automatically do tree shaking.

Reply:
https://github.com/forcedotcom/salesforcedx-vscode/blob/e84a5178d55e911c2ba489598d514d88ad039132/packages/salesforcedx-webview-ui/config/webpack.config.prod.js#L284

Looks like we are still compressing/outputting to ecma5. Let me get it into ecma 6. We should use es6 to be consistent with the rest of the output from vs code. 

I see now that tree shaking is only enabled for es6 or es2015. https://webpack.js.org/guides/tree-shaking/